content_api/new_upload/exceptions.py

```diff
@@ -0,0 +1,62 @@
+class ValidationException(Exception):
+    pass
+
+
+class InvalidUTF8(ValidationException):
```

Comment:
do these work? They inherit from ValidationException, but that doesn't implement a constructor (that calls Exception's constructor)

Reply:
Yeah, this is how Python inheritance works? `ValidationException` is identical to `Exception` in every definition, except for the name of course. So `ValidationException` now has a ctor too, even if it is not defined. Otherwise a lot of Python would break.

This is similar to how C++ works, as far as I know. So I am a bit confused by your question tbh :D